app/enterprise/0.32-x/upgrades-migrations.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Upgrades and Migrations
+---
+
+# Upgrades and Migrations
+## Upgrades of Kong EE
+
+We define an upgrade, the process of changing the version of the software currently deployed to a more recent one, there is no difference with concept of update; normally associated with an upgrade between minor versions.
+
+We suggest to follow the below steps to perform an effective upgrade, this approach will require downtime, in case you need high availability review the Blue/Green Deployment section.
+
+1. Download the destination version of the product you are interest to upgrade to in Bintray
+2. Take a backup copy of the production database
+3. Review the [changelog](https://docs.konghq.com/enterprise/changelog/) for the destination version of the upgrade
+4. Prepare the preproduction environment with the copy of the database taken from production
+5. Install the newer package in preproduction.
+6. Apply any breaking changes to your preproduction environment
+7. Run the command kong migrations up
+8. Start Kong and validate the upgrade in preproduction
+9. Schedule the maintenance window for the upgrade in production
+10. Backup your production database in case of rollback.
+11. Install the package
+12. Adapt your infrastructure, kong configuration file and eventual custom nginx template according with your previous experience in preproduction.
+13. Run the kong migrations up command and start kong
+
+Once a new version is available you will be advised by a distribution list email or via your designated Customer Success Engineer (Premium subscription only). The newer version will be available in the same repository in Bintray.
+Although you may already have backup processes in place, a most recent backup of your database prior upgrade is needed due the fact that will represent the most recent versions of your entities and other metrics which need to be preserved while upgrading, this means that from now on any changes applied via the Admin API wont appear in the destination environment, therefore we suggest to not make modifications on the Admin API during the upgrade process of any is needed you would need to repeat the same in the destination environment after the upgrade. For database backups you can use specific tools for each database such as `pg_dump` for Postgres or `nodetool snapshot` for Cassandra, however we recommend to always refer to an experienced DBA.
```

Comment:
Paragraphs in Markdown are delimited by one or more blank lines. Please format accordingly - as it currently is, this is all a single big chunk of text.

Reply:
The first sentence of this is very long - see if you can break it into two sentences.